pepper-apis/src/main/java/org/broadinstitute/ddp/route/AddProfileRoute.java

```diff
@@ -66,7 +68,7 @@ public Object handle(Request request, Response response, Profile profile) throws
                             .setFirstName(profile.getFirstName())
                             .setLastName(profile.getLastName())
                             .setSexType(sexType)
-                            .setBirthDate(profile.getBirthDate())
+                            .setBirthDate(profile.getBirthDate() != null ? LocalDate.parse(profile.getBirthDate()) : null)
```

Comment:
Since this is a string, I think we should wrap `parse()` in try/catch and return `400` if it's an invalid date.